static/openapi/dem/elevation-openapi.yaml

```diff
@@ -0,0 +1,131 @@
+openapi: 3.0.1
+info:
+  title: dem-heights
+  description: map-colonies service for extracting elevations from DTM/DSM geotiffs
+  version: 1.2.6
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+paths:
+  /points:
+    post:
+      operationId: pointsHeight
+      tags:
+        - heights
+      summary: get points heights
+      requestBody:
+        description: Points to calculate heights for.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/pointsReq'
+          application/octet-stream:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/pointsRes'
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+        '400':
+          description: Points density is too low to compute
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+        '500':
+          description: Service Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
```

Comment:
For some reason the error message displayed is for the `400` statud code.

Reply:
The HyperText Transfer Protocol (HTTP) 400 Bad Request response status code indicates that the server cannot or will not process the request due to something that is perceived to be a client error.

Actually this error in current version will never happen
So, remove it from YAML

Reply:
Can't this happen if the given structure of the point array was built incorrectly?

Reply:
Nope it was relevant only for prev version where we are limit number of requests but not number of points.
"density" term currently is irrelevant(service will not return error 400)